packages/react-i18n/src/index.tsx

```diff
@@ -9,10 +9,10 @@ export interface I18nProps {
 	_n: typeof _n;
```

Comment:
Changes in this package are from https://github.com/Automattic/wp-calypso/pull/39480.